### PR TITLE
Add support for air conditioner AP10TW1RLR-N 006-200

### PR DESCRIPTION
**Pull request by jonchampagne**
Been using this dictionary config for a week or so and it seems to be accurate.

### CODE REVIEW

File: custom_components/connectlife/data_dictionaries/006-200.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+- property: t_sleep
+  icon: mdi:bedtime
+  switch:
+    device_class: switch
+- property: t_super
+  icon: mdi:speed
+  switch:
+    device_class: switch
+# Inlet temperature?
+- property: f_temp_in
```
Comment (oyvindwe) [inline custom_components/connectlife/data_dictionaries/006-200.yaml:62]: This seems to be the current temperatur on other A/C units at least. Can you test this mapping?
```
- property: f_temp_in
  climate:
    target: current_temperature
```

**Review by oyvindwe — CHANGES_REQUESTED** [review]
Thank your very much for your contribution ❤️ 

I left one comment.

File: custom_components/connectlife/data_dictionaries/006-200.yaml (hunk 1 above)
Comment (jonchampagne) [reply custom_components/connectlife/data_dictionaries/006-200.yaml:62]: Thanks,

I don't think this is the current temperature, or at least if it is than it's an inaccurate reading. 
Comparing it with a climate sensor both in the same room and on my balcony, this seems to much more align with the outdoor temperature than the indoor one. I'm not sure though whether this is just down to where the temperature probe is placed within the AC unit or not. 

image

**Review by oyvindwe — CHANGES_REQUESTED** [review]

**Comment by smaddock** [comment]
Interesting, I had just duplicated the 006-201 data dictionary for my 006-200 (almost identical to the one here) which also has f_temp_in mapped as a property, and my only complaint was I couldn’t reference it in sensor cards.

**Comment by smaddock** [comment]
Okay to my previous comment, I set `f_temp_in` as:
```yaml
- property: f_temp_in
  icon: mdi:temp
  climate:
    target: current_temperature
```
And then created a sensor template helper to get me the sensor data I wanted:
```jinja
{{ state_attr("climate.portable_air_conditioner", "current_temperature") }}
```

**Comment by smaddock** [comment]
A few other notes testing your config @jonchampagne:

- `daily_energy_kwh` is always zero on my model, so I'd removed it
- `t_eco` does nothing on my model, I try setting it but the API keeps returning zero, so I'd removed it
- It looks like you're specifying non-core icons for `t_sleep` and `t_super`; you might want to change those to `mdi:power-sleep` and `mdi:fast-forward` respectively just for compatibility
- I'll submit a separate PR to get the Sleep switch added to the i18n options
- `t_fan_mute` is valid on my model, however it is redundant with the built-in "Disable beep" option when you configure a device with this integration, so probably only worth adding if you enable and disable the beep often. If you want to add, it can be set as:
```yaml
- property: t_fan_mute
  icon: mdi:volume-variant-off
  switch:
    device_class: switch
```

**Comment by smaddock** [comment]
@jonchampagne if you can’t or don’t want to make these changes I can submit a new PR as it looks like @Go2Engle also has this model, but I’d rather you get the credit for being the first to submit.

**Comment by oyvindwe** [comment]
Superseeded by #105 

I'll make sure to credit both of you as well in the release notes, @jonchampagne and @smaddock :)

**Pull request closed without merging**